src/omv/sensors/genx320.c

```diff
@@ -49,8 +49,12 @@
 #define ACTIVE_SENSOR_HEIGHT            (SENSOR_HEIGHT - BLANK_LINES)
 #define ACTIVE_SENSOR_SIZE              (ACTIVE_SENSOR_WIDTH * ACTIVE_SENSOR_HEIGHT)
 
-#define HSYNC_CLOCK_CYCLES              32
-#define VSYNC_CLOCK_CYCLES              32
+#ifdef OMV_GENX320_EHC_ENABLE
+#define HSYNC_CLOCK_CYCLES              880 // 320 + 880 = 1200 cycles -> ~122 FPS
+#else
+#define HSYNC_CLOCK_CYCLES              280 // 320 + 280 = 600 cycles
+#endif
+#define VSYNC_CLOCK_CYCLES              8
```

Comment:
Settings per camera type aren't required anymore. These defaults work for the H7 and RT1062 and are loose. set_framerate() then can be called to speed up the FPS and push towards the max of what the system can do.

Reply:
But will they be okay for N6?

Reply:
Should be, the N6 can handle the previous hsync of 32 which was much tighter than 880.